SMLHelper/Options/KeybindModOption.cs

```diff
@@ -15,7 +14,7 @@
     /// <summary>
     /// Contains all the information about a keybind changed event.
     /// </summary>
-    public class KeybindChangedEventArgs : EventArgs
+    public class KeybindChangedEventArgs : EventArgs, Interfaces.IModOptionEventArgs
```

Comment:
Could we get `using Interfaces` up top to clean this up?

Reply:
https://github.com/SubnauticaModding/SMLHelper/pull/173/commits/5aac61157d36e9f5703e88746e8a3b0fe08ef79c